test/algorithms/ground_state_solvers/test_advanced_ucc_variants.py

```diff
@@ -0,0 +1,141 @@
+# This code is part of Qiskit.
```

Comment:
FYI: If someone might wonder where the tests from `test/components/variational_forms` went, this is one new file which contains some of those tests.
The others have been integrated into other places.